src/lib/components/ui/ScrollView/README.mdx

```diff
@@ -0,0 +1,417 @@
+        dis parturient montes, nascetur ridiculus mus. Donec quam felis,
+        ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa
+        quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget,
+        arcu. In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo.
+        Nullam dictum felis eu pede mollis pretium. Integer tincidunt. Cras
+        dapibus. Vivamus elementum semper nisi. Aenean vulputate eleifend
+        tellus. Aenean leo ligula, porttitor eu, consequat vitae, eleifend ac,
+        enim. Aliquam lorem ante, dapibus in, viverra quis, feugiat a, tellus.
+        Phasellus viverra nulla ut metus varius laoreet. Quisque rutrum. Aenean
+        imperdiet. Etiam ultricies nisi vel augue. Curabitur ullamcorper
+        ultricies nisi. Nam eget dui. Etiam rhoncus. Maecenas tempus, tellus
+        eget condimentum rhoncus, sem quam semper libero, sit amet adipiscing
+        sem neque sed ipsum. Nam quam nunc, blandit vel, luctus pulvinar,
+        hendrerit id, lorem. Maecenas nec odio et ante tincidunt tempus. Donec
+        vitae sapien ut libero venenatis faucibus. Nullam quis ante. Etiam sit
+        amet orci eget eros faucibus tincidunt. Duis leo. Sed fringilla mauris
+        sit amet nibh. Donec sodales sagittis magna. Sed consequat, leo eget
+        bibendum sodales, augue velit cursus nunc.Aenean massa. Cum sociis
+        natoque penatibus et magnis dis parturient montes, nascetur ridiculus
+        mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis,
+        sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel,
+        aliquet nec, vulputate eget, arcu. In enim justo, rhoncus ut, imperdiet
+        a, venenatis vitae, justo. Nullam dictum felis eu pede mollis pretium.
+        Integer tincidunt. Cras dapibus. Vivamus elementum semper nisi. Aenean
+        vulputate eleifend tellus. Aenean leo ligula, porttitor eu, consequat
+        vitae, eleifend ac, enim. Aliquam lorem ante, dapibus in, viverra quis,
+        feugiat a, tellus. Phasellus viverra nulla ut metus varius laoreet.
+        Quisque rutrum. Aenean imperdiet. Etiam ultricies nisi vel augue.
+        Curabitur ullamcorper ultricies nisi. Nam eget dui. Etiam rhoncus.
+        Maecenas tempus, tellus eget condimentum rhoncus, sem quam semper
+        libero, sit amet adipiscing sem neque sed ipsum. Nam quam nunc, blandit
+        vel, luctus pulvinar, hendrerit id, lorem. Maecenas nec odio et ante
+        tincidunt tempus. Donec vitae sapien ut libero venenatis faucibus.
+        Nullam quis ante. Etiam sit amet orci eget eros faucibus tincidunt.
+        Duis leo. Sed fringilla mauris sit amet nibh. Donec sodales sagittis
+        magna. Sed consequat, leo eget bibendum sodales, augue velit cursus
+        nunc.
+      </div>
+    </ScrollView>
+  </Placeholder>
+</Playground>
+
+## General Guidelines
```

Comment:
Why don't we add information about autoscroll in general guidelines? Something like: If you are implementing component which content is dynamic (e.g. chat window, network monitoring console, ..), you should enable autoscroll to present latest data to users. Or something like that

Reply:
Didn't think of it when I was rebasing the docs after this feature had been implemented. I'm adding it now.